lib/org/chromium/webidl/PipelineRunner.js

```diff
@@ -36,7 +37,8 @@ foam.CLASS({
       return new Error(`${this.cls_.name}: ${msg}`);
     },
     function validateMessage(message) {
-      if (!this.PipelineMessage.isInstance(message)) {
+      if (!this.PipelineMessage.isInstance(message) &&
+          !this.CanonicalCollection.isInstance(message)) {
```

Comment:
This is temporary. Once `PipelineMessage` is removed, this will check to see if `message` is an instance of `inputType` instead